script.py

```diff
@@ -42,11 +46,17 @@
     show_phone_number = driver.find_elements_by_css_selector(SHOW_PHONE_NUMBER_CSS)
     if show_phone_number:
         show_phone_number[1].click()
-        phone_number = driver.find_elements_by_css_selector(PHONE_NUMBER_CSS)[1].text
+        # Todo: add 2+ numbers case
+        if show_phone_number[1].get_attribute('style') == PHONE_NUMBER_VISIBLE:

```

Comment:
Make it easier
        if show_phone_number[1].get_attribute('style') != PHONE_NUMBER_VISIBLE:
            WebDriverWait(driver, 10).until(element_has_attribute(
                (By.CSS_SELECTOR, SHOW_PHONE_NUMBER_CSS), 'style', PHONE_NUMBER_VISIBLE))

        phone_number = driver.find_elements_by_css_selector(PHONE_NUMBER_CSS)[1].text